VideoWeb/VideoWeb/ClientApp/src/assets/i18n/cy.json

```diff
@@ -485,6 +485,7 @@
     "are-not-allowed-to-smoke-including-e-cigarettes": "ni chaniateir ichi ysmygu (gan gynnwys e-sigarets)",
     "must-be-alone-unless-you-have-the-judges-permission": "rhaid ichi fod ar ein pen ein hun oni bai bod y barnwr wedi rhoi caniatâd ichi gael rhywun gyda chi",
     "must-ask-the-judges-permission-to-move-away-from-screen": "rhaid ichi ofyn am ganiatâd y barnwr os byddwch angen symud i ffwrdd o'ch sgrin ",
+    "abuse-warning": "abuse will NOT be tolerated, anyone using threatening or aggressive behaviour against our staff or judges will  be removed from the hearing. You may also be reported to and arrested by the Police; you may be sent to Prison; your case may continue and be decided in your absence.",
```

Comment:
could we get the welsh for this?

Reply:
Michael is working on it. We will have a different ticket to update the Welsh